tools/stubs/logging.pyi

```diff
@@ -0,0 +1,22 @@
+class Formatter:
```

Comment:
Perhaps add a little file docstring to let developers know this is only to take into account the `.trace()` method we manually add at the runtime.